l1-contracts/terraform/main.tf

```diff
@@ -109,4 +109,13 @@ variable "GOVERNANCE_CONTRACT_ADDRESS" {
 
 output "GOVERNANCE_CONTRACT_ADDRESS" {
   value = var.GOVERNANCE_CONTRACT_ADDRESS
-}
\ No newline at end of file
+}
+
+variable "SLASH_FACTORY_CONTRACT_ADDRESS" {
```

Comment:
side note, I'm quite sure this terraform is dead code.

Reply:
@alexghr or @spypsy, can you comment?

Reply:
Should we just get rid of them then? If not used